Remotion/Web/Core/UI/SmartPageImplementation/SmartPageInfo.cs

```diff
@@ -487,7 +487,7 @@ private void FormatPopulateTrackedControlsArrayClientScript (StringBuilder scrip
         }
       }
 
-      foreach (string trackedID in _trackedControlsByID)
+      foreach (string? trackedID in _trackedControlsByID)
```

Comment:
should not be nullable

Reply:
As `StringCollection` enumerates through `string?`, that's the type here. I can't think about any adaptation that would make it prettier. Could add `.Cast<string>()` if you want.